### ISSUE TITLE
Customizations in SyntaxPad 🖌️🎨

### ISSUE THREAD

**Issue by kothariji** [kothariji/SyntaxMeets#26]
## Description
To add Customisation options in syntaxpad like

- increase/decrease brush size.
- eraser
- adding lines/curves
- adding shapes like circle/ box.
- changing background-color

**Comment by Vishv07** [comment]
Well. I feel React-canvas-draw isnt good choice.  [Though it looks good🔥⚡]
It has very less features and adding some Customisation in this will be bit complex.


Here are some demos you guys should have a look

1) http://mikhail-angelov.github.io/react-whiteboard/

2) https://react-drawing-board-demo-dilidili.vercel.app/

3) https://whiteboard-socketio.herokuapp.com/  [Look at this , this one is Collabrative/Sharable ]

3rd One With some changes can fix  #22  👍👍

**Comment by kothariji** [comment]
True. there are limitations in react-canvas-draw. We aren't provided with much functionalities. 

These suggestions look simply amazing.
We we definitely look forward into those applications. 
If you wish to work on this issue, let me know for the same 🙌🔥

**Comment by Vishv07** [comment]
I want to work on this issue as a part of GSSOC'21

**Comment by micksharma** [comment]
I want to work on this issue.

**Comment by kothariji** [comment]
> I want to work on this issue as a part of GSSOC'21

@Vishv07 Great 🚀
I am assigning this issue to you 👍

**Comment by Akash-Salvi** [comment]
Hey, @micksharma, thank you for showing interest 🚀

We appreciate🙏 your dedication and efforts towards our project, but according to our [**contributing guidelines**](https://github.com/kothariji/SyntaxMeets/blob/master/CONTRIBUTING.md), Issues are assigned on the **First Come First Serve** basis.

Anyways feel free to check out other **unassigned issues**, or create an Issue if you spot one🎯.

Thank you😊, and Sorry for the inconvenience caused 🙌

**Comment by Priyankcoder** [comment]
> Well. I feel React-canvas-draw isn't a good choice. [Though it looks good]
> It has very few features and adding some Customizationin in this will be a bit complex.
> 
> Here are some demos you guys should have a look
> 
> 1. http://mikhail-angelov.github.io/react-whiteboard/
> 2. https://react-drawing-board-demo-dilidili.vercel.app/
> 3. https://whiteboard-socketio.herokuapp.com/  [Look at this, this one is Collaborative/Sharable ]
> 
> 3rd One With some changes can fix #22

@Vishv07 Here is a feature suggestion, Please make the whiteboard infinite and scrollable. That way users can write their mind out without caring about space.
You can take the inspiration from here: https://miro.com/app/dashboard/

**Comment by Vishv07** [comment]
@Priyankcoder  Thanks, mate! I Will surely look into this

**Comment by Vishv07** [comment]
@Priyankcoder This is next Level 🔥🔥🔥🔥 [https://miro.com/app/dashboard/]

Do you know any open source lib which has kinda same features

**Comment by Priyankcoder** [comment]
@Vishv07 I don't know about that, but I found some results on searching.
[1](https://github.com/daniellowtw/infBoard), [2](https://github.com/renrut/Infinite-Whiteboard), and [3](https://github.com/TomHumphries/InfiniteCanvasWhiteboard).

**Comment by Vishv07** [comment]
Started working on this @kothariji
